ui/src/utils/helper.ts

```diff
@@ -66,3 +72,35 @@ export const formatPatchPoolData = (originUnitData: PoolDetailType,type:'edit'|'
   }
   return newOriginUnitData;
 };
+
+export const getAppInfoFromStorage = async (): Promise<API.AppInfo> => {
```

Comment:
What if the version changes?

Reply:
SessionStorage will be cleared when the window is closed or the browser is closed. Do users need to build different packages when using different versions? If it is, there will be no impact

Reply:
I see, storing in sessionStorage is OK.